activesupport/test/number_helper_test.rb

```diff
@@ -344,6 +344,9 @@ def test_number_to_human_with_custom_units
           #Spaces are stripped from the resulting string
           assert_equal "4", number_helper.number_to_human(4, units: { unit: "", ten: "tens " })
           assert_equal "4.5  tens", number_helper.number_to_human(45, units: { unit: "", ten: " tens   " })
+
+          #Uses only provided units and not larger ones
```

Comment:
Thanks for opening up this PR @sai29! Can you add a space here between the `#` and `Uses`?